### PR TITLE
update 3rd Party.rst

### PR DESCRIPTION
**Pull request by dragons8mycat**
Make grammatical and spelling changes

### CODE REVIEW

File: source/docs/user_manual/processing/3rdParty.rst [hunk 1]
```diff
@@ -387,7 +387,7 @@ in your system and you have configured |qg| properly, so it can find all
 necessary files (command-line tools and libraries).
 
 As in the case of SAGA, OTB binaries are included in the stand-alone installer for
-Windows, but they are not included if you are runing Linux, so you have to download
+Windows, but they are not included if you are ruining Linux, so you have to download
```
Comment (yjacolin) [inline source/docs/user_manual/processing/3rdParty.rst]: not sure if ruining is the right term :) running seems more appropriate to me ;)

**Comment by yjacolin** [comment]
looks good exept ruining to be changed to running!

**Comment by dragons8mycat** [comment]
Haha...stoopid spallchack :P 
...though it does have a point..."but they are not included if you are ruining Linux"

**Comment by dragons8mycat** [comment]
What is the best method to correct this without causing a crash? Should I correct it in the branch and then re-send the pull request?

**Comment by yjacolin** [comment]
just:
vim thefile.rst
git add thefile.rst
git commit --amend
git push --force origin mybranch

The --amend allow you to change (or not) the commit message, but you can just quit the vim interface without change anything.

The last command push  --force should be used only in your branch if you are the only one to work on, which is the case here.

That's all, PR will be updated and my comment show as deprecated.

**Comment by dragons8mycat** [comment]
*groan* I accidentally committed the "commander.rst" update direct to master somehow, I was editing it in branch!! How can I fix this?

**Comment by yjacolin** [comment]
not sure, but youc an try: git revert HEAD

File: source/docs/user_manual/processing/batch.rst [hunk 2]
```diff
@@ -81,13 +81,13 @@ For most parameters, setting the value is trivial. Just type the value or select
 it from the list of available options, depending on the parameter type.
 
 Filenames for input data objects are introduced directly typing or, more
-conveniently, clicking on the |browsebutton| button on the right hand of the cell, wich will show a context menu with two option: one for selecting from the layers currently opened and another to secet from the filesystem. This second option, when selected, shows a typical file chooser dialog. Multiple files can be selected at once.
+conveniently, clicking on the |browsebutton| button on the right hand of the cell, which will show a context menu with two option: one for selecting from the layers currently opened and another to select from the filesystem. This second option, when selected, shows a typical file chooser dialog. Multiple files can be selected at once.
```
Comment (DelazJ) [inline source/docs/user_manual/processing/batch.rst]: two option**s**. Then can you split this line, please? too long... maybe options description can be put in a bulleted list to better distinguish them.

File: source/docs/user_manual/processing/batch.rst [hunk 3]
```diff
@@ -81,13 +81,13 @@ For most parameters, setting the value is trivial. Just type the value or select
...
 If the input parameter represents a single data object and several files are
 selected, each one of them will be put in a separate row, adding new ones if
 needed. If the parameter represents a multiple input, all the selected files will be added
 to a single cell, separated by semicolons (``;``).
 
-Layer identifiers can be directly typed in the parameter teext box. You can enter the full path to a file or the name of a layer that is currently loaded in the current QGIS project. The name of the layer will be automatically resolved to its source path. Notice that, if several layers have the same name, this might cause unexpected results due to ambiguity.
+Layer identifiers can be directly typed in the parameter text box. You can enter the full path to a file or the name of a layer that is currently loaded in the current QGIS project. The name of the layer will be automatically resolved to its source path. Notice that, if several layers have the same name, this might cause unexpected results due to ambiguity.
```
Comment (DelazJ) [inline source/docs/user_manual/processing/batch.rst]: another long line to split

File: source/docs/user_manual/processing/3rdParty.rst [hunk 4]
```diff
@@ -563,7 +563,7 @@ two instances of each tool in toolbox and can use them in your analysis.
 
 **IMPORTANT!** Be careful with developing Processing models using TauDEM. As
 single- and multifile versions have different inputs, model created with
-singlefile algortihms will not work if only multifile algorithms are available.
+singlefile algorithms will not work if only multifile algorithms are available.
```
Comment (DelazJ) [inline source/docs/user_manual/processing/3rdParty.rst]: single file or singlefile? same for multi

**Pull request merged**